src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get condition recommendations
+metaDescription: For APM entities, you can see and create recommended alert conditions.
+watermark: Limited release
+---
+
+Use the New Relic Navigator's mini-overview to get alert condition recommendations for your monitored entities. With these recommendations, you can create alerts conditions to be notified when your entities aren't performing as they should.
+
+<Callout variant="caution">
```

Comment:
I wonder if a caution callout is overkill? How about <important>?

Reply:
Yeah, a tip callout might ok here.